app/main.py

```diff
@@ -2,0 +1,24 @@
+import os
+
+
+def copy_file(command: str) -> None:
+    command_parts = command.split()
+
+    if command_parts[1] == command_parts[2]:
+        return
+
+    if command_parts[0] != "cp":
+        print("Wrong command use 'cp'.")
+        return
+
+    if not os.path.exists(command_parts[1]):
+        print(f"File {command_parts[1]} not exist.")
+        return
+
+    try:
+        with (open(command_parts[1], "r") as source_file,
+              open(command_parts[2], "w") as destination_file):
```

Comment:
Code Style: The variable names `source_file` and `destination_file` are not descriptive enough. Consider renaming them to something like `source_filename` and `destination_filename` for better code readability.